pyproject.toml

```
@@ -1,21 +1,48 @@
 [tool.poetry]
 name = "multisignal-epi-inference"
 version = "0.1.0"
-description = "Python package for statistical inference and forecast of epi models using multiple signals."
+description = "CFA Multisignal Renewal project develops Pyrenew, a Python package for multi-signal Bayesian renewal modeling with JAX and Numpyro."
```

This is a  bit clunky to me. Instead of develops, maybe something like
```suggestion
description = "The CFA Multisignal Renewal project contains Pyrenew, a Python package for multi-signal Bayesian renewal modeling with JAX and Numpyro."
```